apps/web/src/styles/theme.ts

```diff
@@ -159,6 +159,18 @@ const theme = extendTheme({
         },
       },
       variants: {
+        dropdownOption: {
+          display: "flex",
+          justifyContent: "flex-start",
+          w: "full",
```

Comment:
please avoid using these shorthands. it's too easy to make a mistake with those.